changelog.d/gh-8408.added

```diff
@@ -0,0 +1,33 @@
+Added support in Ruby, Julia, and Rust to match implicit return statement inside functions.
+
+For example:
+
+```julia
+return 0
+```
+
+can now match 0 in
+
+```julia
+function f()
+  0
+end
+```
+
+This matching can be disabled with the rule option `implicit_return`.
+
+For example:
```

Comment:
I think you don't need this example. But we would need to update the docs.

Reply:
Ok I'll remove it.